Source/ASNetworkImageNode.mm

```diff
@@ -29,19 +30,21 @@
 #import <AsyncDisplayKit/ASLog.h>
 #import <AsyncDisplayKit/ASNetworkImageLoadInfo+Private.h>
 
+#import <atomic>
+
 #if AS_PIN_REMOTE_IMAGE
 #import <AsyncDisplayKit/ASPINRemoteImageDownloader.h>
 #endif
 
 @interface ASNetworkImageNode ()
 {
-  // Only access any of these with __instanceLock__.
+  // Only access any of these while locked.
   __weak id<ASNetworkImageNodeDelegate> _delegate;
 
   NSURL *_URL;
   UIImage *_defaultImage;
 
-  NSUUID *_cacheUUID;
+  std::atomic_uint _cacheToken;
```

Comment:
We may should have this change in a follow up diff. Not really 100% related to adopting NSLocking.